lib/location/location_core.c

```diff
@@ -495,6 +514,23 @@ void location_core_event_cb(const struct location_data *location)
 	}
 }
 
+void location_core_event_cb_fn(struct k_work *work)
+{
+	location_core_event_cb_handle();
```

Comment:
Maybe this function can be removed and `location_core_event_cb_handle()` changed to a work instead?